### DISCUSSION TITLE
npx create-sa-addon

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#2954] [Show and tell]
https://github.com/TheColaber/create-sa-addon 

That's right, now you no longer need to create your own addon folder and stuff. Try it out!
![image](https://user-images.githubusercontent.com/72760579/125162923-9f3ed980-e158-11eb-80db-68445e9e4bea.png)
This produces: 
![image](https://user-images.githubusercontent.com/72760579/124968032-270ad380-dff3-11eb-8ed1-39e541a5a57e.png)
![image](https://user-images.githubusercontent.com/72760579/124968058-2bcf8780-dff3-11eb-85cc-8eea371102e6.png)
![image](https://user-images.githubusercontent.com/72760579/124968075-2ffba500-dff3-11eb-87d3-1a616cd98b14.png)
![image](https://user-images.githubusercontent.com/72760579/124968088-35f18600-dff3-11eb-938e-0ac2d92412a9.png)

So all you got to do is `npx create-sa-addon`!

**Comment by cobaltt7** [comment]
Please note that you will need to have [node installed](https://nodejs.org/en/) for this to work.

**Comment by pufferfish101007** [comment]
This is really cool! A couple of mistakes though:
 - seperate should be separate
 - userstyle says 'leave empty for no user**script**'
